File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.cpp
// Last updated: 3/13/2026, 8:16:22 PM
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int result = 0;
        for(string st : operations){
            result+= st[1] == '+' ? 1 : -1;
        }
        return result;
    }
};

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.java
// Last updated: 3/13/2026, 8:16:23 PM
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int result = 0;
        for(String str : operations){
            if(str.contains("-")) result--;
            else result++;
        }
        return result;
    }
}